api/requester/responses.go

```diff
@@ -22,15 +22,15 @@ type rpcResponse struct {
 }
 
 type seedResponse struct {
-	Seed    []byte `json:"Seed"`
+	Seed    string `json:"Seed"`
 	TraceID string `json:"TraceID"`
 }
 type rpcSeedResponse struct {
 	rpcResponse
 	Result seedResponse `json:"result"`
 }
 
-// StatusResponse represents response from rpc on status.Get method
+// StatusResponse represents response from rpc on node.GetStatus method
 type StatusResponse struct {
 	NetworkState string `json:"NetworkState"`
```

Comment:
Why the first letter is in the upper case? In other places, you use `networkState` format.